CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+cmake_minimum_required(VERSION 3.1)
+project(node-osrm C CXX)
+
+# Explicitly set the build type to Release if no other type is specified
+# on the command line.  Without this, cmake defaults to an unoptimized,
+# non-debug build, which almost nobody wants.
+if(NOT CMAKE_BUILD_TYPE)
+  message(STATUS "No build type specified, defaulting to Release")
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+option(BUILD_LIBOSRM "Download and build own libsorm version" OFF)
+
+set(OSRM_BINARIES "")
+if(BUILD_LIBOSRM)
+  set(OSRM_INSTALL_DIR "/tmp/osrm-backend")
```

Comment:
Hrm why do we put it there? Asking since it may clash with existing osrm-backend dirs there.